crates/subspace-farmer/src/plot/piece_index_hash_to_offset_db.rs

```diff
@@ -54,61 +54,129 @@ impl IndexHashToOffsetDB {
     /// You can find discussion of derivation of this number here:
     /// https://github.com/subspace/subspace/pull/449
     const MAX_DISTANCE_CACHE_ONE_SIDE_LOOKUP: usize = 8000;
-    const METADATA_COLUMN_FAMILY: &'static str = "metadata";
-    const PIECE_COUNT_KEY: &'static str = "piece_count";
+    const METADATA_COLUMN: u8 = 0;
+    const DATA_COLUMN: u8 = 1;
+    const PIECE_COUNT_KEY: &'static [u8] = b"piece_count";
+
+    fn options(path: PathBuf) -> Options {
```

Comment:
I just use it twice in both general `open` method and in the migration. Once migration will be removed, it can be inlined.

Reply:
Not blocking, but in such cases I prefer copy-pasting, so that you can delete code right away instead of remembering that after deleting temporary code you need to go elsewhere and do extra refactoring.

Reply:
It would require to change options in several places if we decide to change them.